agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wascore/AppInsightsByteBufferPool.java

```diff
@@ -19,7 +19,7 @@
  * DEALINGS IN THE SOFTWARE.
  */
 
-package com.microsoft.applicationinsights;
+package com.microsoft.applicationinsights.agent.internal.wascore;
```

Comment:
you created wascore? it can be anything right?

Reply:
ya, I promise I will fix both `wascore` and `wasbootstrap` soon